packages/fxa-payments-server/src/components/PaymentConsentCheckbox/en.ftl

```diff
@@ -1,3 +1,3 @@
 ## Component - PaymentConsentCheckbox
 
-payment-confirm-with-legal-links-static = I authorize { -brand-name-mozilla }, maker of { -brand-name-firefox } products, to charge my payment method for the amount shown, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.
+payment-confirm-with-legal-links = I authorize { -brand-name-mozilla }, maker of { -brand-name-firefox } products, to charge my payment method {$price} for {$productName}, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.
```

Comment:
```suggestion
# Variables: $price (String) - The amount of the subscription being billed, including currency, e.g. $10.00. 
# $productName (String) - The name of the subscribed product, e.g. Mozilla VPN.
payment-confirm-with-legal-links = I authorize { -brand-name-mozilla }, maker of { -brand-name-firefox } products, to charge my payment method { $price } for { $productName }, according to <termsOfServiceLink>Terms of Service</termsOfServiceLink> and <privacyNoticeLink>Privacy Notice</privacyNoticeLink>, until I cancel my subscription.
```
Please include information in comments for what and how variables will be rendered in the string.
Variables in fluent should also have a space before and after the variable name within the brackets.